integrate/artifacts/tutorial.md

```diff
@@ -0,0 +1,152 @@
+    -H "X-StorageApi-Token: $TOKEN" \
+    -H 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'configuration={"parameters":{"blocks":[{"name":"Block 1","codes":[{"name":"artifacts","script":["import os\nimport glob\n\n# Download\nprint(glob.glob(\"/data/artifacts/in/runs/*/*\")) \n\n# Upload\npath = \"/data/artifacts/current\"\nif not os.path.exists(path):os.makedirs(path)\nwith open(\"/data/artifacts/in/current/myartifact1\", \"w\") as file:\n  file.write(\"value1\")"]}]}]},"artifacts":{"runs":{"enabled":true,"filter":{"limit":5}}}}' \
+    --data-urlencode 'name=Artifacts upload & download' \
+    --data-urlencode 'description=Test Artifacts upload & download'
+    ```
+   
+4. Here's the configuration related to artifacts from the previous command:
+    This will enable download of artifacts of type `runs` with limit 5, which means this will download artifacts created by the last 5 runs of the same component configuration
+    ```json
+    {
+      "artifacts":{
+        "runs":{
+          "enabled":true,
+          "filter":{
+            "limit":5
+          }
+        }
+      }
+    }
+    ```
+5. The script from the configuration will read files from `/data/artifacts/in/runs/*/*` and write them to the output - these are artifact files downloaded. 
+   The script will also generate a new artifact and write it to `/data/artifacts/in/current/myartifact1` as in previous example.
+
+    ```python
+    import os
+    import glob
+   
+    # Download
+    print(glob.glob("/data/artifacts/in/runs/*/*")) 
+   
+    # Upload
+    path = "/data/artifacts/out/current"
+    if not os.path.exists(path):os.makedirs(path)
+    with open("/data/artifacts/out/current/myartifact1", "w") as file:
+      file.write("value1")
+    ```
+
+## Example 3 - Consume artifacts from different component
+Similar to previous example we will create a configuration of Python Transformation component. 
+But this time we will download artifacts produced by the configuration from `Example 2`.
+
+1. Export the id of the previously created configuration into an environment variable:
+    ```shell
+    export CONFIG_ID="..."
+    ```
+   
+2. Run curl command
+    ```shell
+    curl -X POST "$STORAGE_API_HOST/v2/storage/branch/default/components/keboola.python-transformation-v2/configs" \
+    -H "X-StorageApi-Token: $TOKEN" \
+    -H 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'configuration={"parameters":{"blocks":[{"name":"Block 1","codes":[{"name":"artifacts","script":["import os\nimport glob\n\n# Download\nprint(glob.glob(\"/data/artifacts/in/custom/*/*\"))"]}]}]},"artifacts":{"custom":{"enabled":true,"component_id":"keboola.python-transformation","config_id":"$CONFIG_ID","branch_id":"default","filter":{"limit":5}}}}' \
+    --data-urlencode 'name=Artifacts upload & download' \
+    --data-urlencode 'description=Test Artifacts upload & download'    
+    ```
+   
+3. The whole configuration now looks like this:
```

Comment:
dtto - 1. a 2. jsou pokyny, co udelat, tohle uz jen popisuje vysledek, tak by to mel byt nomrlani odstavec.